third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1248,6 +1246,7 @@ static void HandleBrowserEvent(const char* name, const char* payload) {
   if (!gBrowserEvents) {
     gBrowserEvents = new std::vector<BrowserEvent>();
   }
+  fprintf(stderr, "KVKV-HandleBrowserEvent - %s - %s\n", name, payload);
```

Comment:
debugging leftover?

Reply:
Yeah, but I left it in explicitly.  It won't hurt anything as that code's not exercised in release product ATM, and it helps debug so it's fine to keep it in tree and clean it up when it becomes less useful.